<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClientAccountPad</class>
 <widget class="QWidget" name="CClientAccountPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblUstID">
     <property name="text">
      <string>Ust.-ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horiUstID">
     <item>
      <widget class="QLineEdit" name="txtUstID"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblMyAccRef">
     <property name="text">
      <string>Eigene Kd.Nr.:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horiMyAccRef">
     <item>
      <widget class="QLineEdit" name="txtMyAccRef"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblBankAccs">
     <property name="text">
      <string>Bank-Konten:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableWidget" name="tbvwBankAccounts">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bankverbindungen:&lt;/p&gt;&lt;p&gt;&amp;quot;S-&amp;gt;&amp;quot; kennzeichnet die Standard-Verbindung.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier können Sie Bankverbindungen hinterlegen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Neu:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Beginnen Sie in der freien Zeile einfach mit der Eingabe.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Ändern:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Überschreiben Sie die entsprechenden Felder.&lt;/p&gt;&lt;p&gt;Sobald Sie ein Feld verlassen haben, wird dessen Inhalt gespeichert.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Standard-Verbindung festlegen:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Drücken Sie &lt;span style=&quot; font-style:italic;&quot;&gt;Alt+S&lt;/span&gt;, um die Standardverbindung festzulegen.&lt;/p&gt;&lt;p&gt;Alternativ können Sie die gewünschte Verbindung mit der rechten Maustaste anklicken und aus dem erscheinenden Menü &lt;span style=&quot; font-style:italic;&quot;&gt;&amp;quot;Als Standard setzen&amp;quot;&lt;/span&gt; auswählen&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Löschen:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Drücken Sie die Taste &lt;span style=&quot; font-style:italic;&quot;&gt;&amp;quot;Entf.&amp;quot;&lt;/span&gt;, um die aktuell markierte Verbindung zu löschen.&lt;/p&gt;&lt;p&gt;Alternativ können Sie die Verbindung mit der rechten Maustaste anklicken und aus dem erscheinenden Menü &lt;span style=&quot; font-style:italic;&quot;&gt;&amp;quot;Löschen...&amp;quot; &lt;/span&gt;auswählen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtUstID</tabstop>
  <tabstop>txtMyAccRef</tabstop>
  <tabstop>tbvwBankAccounts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
